static/app/components/search/searchResultWrapper.tsx

```diff
@@ -1,17 +1,7 @@
 import {css} from '@emotion/react';
 import styled from '@emotion/styled';
 
-interface Props extends React.HTMLAttributes<HTMLDivElement> {
-  highlighted?: boolean;
-}
-
-function scrollIntoView(element: HTMLDivElement) {
-  element?.scrollIntoView?.({block: 'nearest'});
```

Comment:
Wait but how does it scroll with keyboard navigation then?

IMO Having keyboard navigation scrolling is more important than fixing some jitter from the mouse hover.

If we want to fix this, we should try and keep the auto-scrolling behavior when navigating with the keyboard (in my opinion at least)

Reply:
`Autocomplete` handles the scrolling logic, which was added in https://github.com/getsentry/sentry/pull/38809

Looks like this was something I should have removed when I made that change but missed

Reply:
Awesome!